src/nrnpython/nrnpy_p2h.cpp

```diff
@@ -832,10 +826,9 @@ static PyObject* py_allgather(PyObject* psrc) {
 
 static PyObject* py_gather(PyObject* psrc, int root) {
     int np = nrnmpi_numprocs;
-    size_t sz;
-    char* sbuf = pickle(psrc, &sz);
+    auto sbuf = pickle(psrc);
     // what are the counts from each rank
-    int scnt = int(sz);
+    int scnt = sbuf.size();
```

Comment:
I don't know if NRN prefers special handling for this, e.g. `static_cast<int>(...)`.

Reply:
I did for all